web_action_conditionable/static/src/js/views.js

```diff
@@ -0,0 +1,23 @@
+/*global openerp, _, $ */
+
+openerp.web_action_conditionable = function (instance) {
+  instance.web.View.include({
+    is_action_enabled: function(action) {
+      var attrs = this.fields_view.arch.attrs;
+      if (action in attrs) {
+	try {
+	  data = JSON.parse(attrs[action]);
+	  return data;
+	} catch(error) {
```

Comment:
@hbrunn attrs comes with "true" when developer puts explicit and this module wants to support simple expression to decide.
parse  `state=='draft'` gives syntax error